variables.tf

```diff
@@ -143,6 +143,12 @@ variable "time_limit_ms" {
   type        = number
 }
 
+variable "assets_update_period" {
```

Comment:
Could we have the variable named assets_update_period_days to have the input just be a number, to avoid bad user input.

Another option would be to do the validation as here https://github.com/snowplow-devops/terraform-aws-snowflake-loader-ec2/blob/master/variables.tf#L159-L162